CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
-
+* Original gesture settings should be maintained after map operations (such as panning the map) complete. ([1989](https://github.com/mapbox/mapbox-maps-android/pull/1989))
```

Comment:
```suggestion
## Features ✨ and improvements 🏁
* Original gesture settings should be maintained after map operations (such as panning the map) complete. ([1989](https://github.com/mapbox/mapbox-maps-android/pull/1989))
```

Reply:
I think it was a `bug` reported by customer, rather than Improvement. but its upto @paulrenn67 😉

Reply:
@ank27 should the changelog entry be marked differently if it is a customer bug fix?

Reply:
Yeah. lets put it in `bug-fix` section.